src/LSSM-Menu.vue

```diff
@@ -62,7 +62,12 @@
             </li>
             <li role="presentation">
                 <a class="lightbox-open" href="https://status.lss-manager.de/">
-                    Server Status
+                   LSSM-Server Status
+                </a>
+            </li>
+            <li role="presentation">
+                <a class="lightbox-open" href="https://stats.uptimerobot.com/3OVrETmBrM/786124522">
```

Comment:
This one is only for de_DE and the button is all games after you add that. 
You can use this link: https://stats.uptimerobot.com/OEKDJSpmvK for all games.

Reply:
oh yeah thx, I was just about to add the missing pages to my status page :)
Thx for the link